src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
![image](https://github.com/mate-academy/layout_stars/assets/96409563/5e4460e1-6eb4-4f94-b0d2-6035b9897b61)
Make sure that the last elements of the row don't have the right margin

Reply:
not fixed